project-koku/koku
# DRAFT: Currency support order

Fixes [COST-####](https://issues.redhat.com/browse/COST-####)

Changes proposed in this PR:
*
*

Testing Instructions:
1. Run `this command` ...
2. Ingest this data `with this other command` ...
3. Expect to see this result:
   ```
   GET http://localhost:8000/api/endpoint

   {json: response}
   ```
4.
5.

---
Additional Context:



---
http://127.0.0.1:8000/api/cost-management/v1/reports/openshift/compute/?filter[resolution]=monthly&filter[time_scope_value]=-1&filter[time_scope_units]=month&group_by[node]=*&order_by[usage]=desc&limit=8

http://127.0.0.1:8000/api/cost-management/v1/reports/openshift/compute/?order_by[usage]=desc&group_by[node]=*&limit=115
